guacamole-docker/README.md

```diff
@@ -163,6 +163,63 @@ The process for doing this via the `mysql` utility included with MySQL is
 documented in
 [the Guacamole manual](http://guacamole.apache.org/doc/gug/jdbc-auth.html#jdbc-auth-mysql).
 
+Deploying Guacamole with SQLServer authentication
+--------------------------------------------------
+
+    docker run --name some-guacamole --link some-guacd:guacd \
+        --link some-sqlserver:sqlserver      \
+        -e SQLSERVER_DATABASE=guacamole_db  \
+        -e SQLSERVER_USER=guacamole_user    \
+        -e SQLSERVER_PASSWORD=some_password \
+        -e SQLSERVER_DATABASE_FILE=/run/secrets/<secret_name> \
+        -e SQLSERVER_USER_FILE=/run/secrets/<secret_name> \
+        -e SQLSERVER_PASSWORD_FILE=/run/secrets/<secret_name> \
+        -d -p 8080:8080 guacamole/guacamole
+
+Linking Guacamole to SQLServer requires three environment variables. If any of
+these environment variables are omitted, you will receive an error message, and
+the image will stop:
+
+1. `SQLSERVER_DATABASE` - The name of the database to use for Guacamole
+   authentication.
+2. `SQLSERVER_USER` - The user that Guacamole will use to connect to SQLServer.
+3. `SQLSERVER_PASSWORD` - The password that Guacamole will provide when
+   connecting to SQLServer as `SQLSERVER_USER`.
+4. `SQLSERVER_DATABASE_FILE` - The path of the docker secret containing the name
+   of database to use for Guacamole authentication.
+5. `SQLSERVER_USER_FILE` - The path of the docker secret containing the name of
+   the user that Guacamole will use to connect to SQLServer.
+6. `SQLSERVER_PASSWORD_FILE` - The path of the docker secret containing the
+   password that Guacamole will provide when connecting to SQLServer as
+   `SQLSERVER_USER.
```

Comment:
I think this section is a bit confusing, as it says that SQLServer authentication "requires three variable" and goes on to list 6. I know you copied the sections above, for MySQL and PostgreSQL, and those need to be separately reviewed and reworked, but it seems like it may make more sense to leave lines 179-187 as-is, and then put a break, and say something like, "Alternatively, if you want to store database credentials using Docker secrets, the following three variables are required and replace the previous three:", and then list the three _FILE variables.

Reply:
Yes you're right, I was just following the other examples. I've updated the text with your instructions.